Octree/include/CGAL/Octree.h

```diff
@@ -0,0 +1,646 @@
+// Copyright (c) 2007-2020  INRIA (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s)     : Jackson Campolattaro, Cédric Portaneri, Tong Zhao
+
+#ifndef CGAL_OCTREE_3_H
+#define CGAL_OCTREE_3_H
+
+#include <CGAL/license/Octree.h>
+
+#include <CGAL/Octree/Node.h>
+#include <CGAL/Octree/Split_criterion.h>
+#include <CGAL/Octree/Walker_criterion.h>
+#include <CGAL/Octree/Walker_iterator.h>
+
+#include <CGAL/bounding_box.h>
+#include <CGAL/Aff_transformation_3.h>
+#include <CGAL/aff_transformation_tags.h>
+
+#include <CGAL/Orthogonal_k_neighbor_search.h>
+#include <CGAL/Search_traits_3.h>
+#include <CGAL/Search_traits_adapter.h>
+#include <CGAL/intersections.h>
+#include <CGAL/squared_distance_3.h>
+
+#include <boost/function.hpp>
+#include <boost/iterator/iterator_facade.hpp>
+#include <boost/range/iterator_range.hpp>
+
+#include <iostream>
+#include <fstream>
+#include <ostream>
+#include <functional>
+
+#include <stack>
+#include <queue>
+#include <vector>
+#include <math.h>
+
+using namespace std::placeholders;
+
+namespace CGAL {
+
+namespace Octree {
+
+/*!
+ * \ingroup PkgOctreeClasses
+ *
+ * \brief Class Octree is a data structure for efficient computations in 3D space.
+ *
+ * \details It builds a heirarchy of nodes which subdivide the space based on a collection of points.
```

Comment:
heirarchy -> hierarchy 
subdivide -> subdivides

Reply:
I fixed the typo in "hierarchy", but I think "subdivide" might be correct. I guess it could be interpreted either way:
"the (singular) hierarchy of nodes subdivides the space" or "a hierarchy consists of (plural) nodes which subdivide the space."